### PR TITLE
Expose primary and secondary pool

### PR DESCRIPTION
**Pull request by aardschok**
In order to reduce the artists time look up current submission settings we added the primary and secondary pool to the `renderglobalsDefault`. Added backwards compatibility.

This PR is in conjunction with : [Maya Deadline Submission tool](https://github.com/Colorbleed/maya-deadline-submission-settings/pull/11)

### CODE REVIEW

File: colorbleed/plugins/global/load/copy_file.py [hunk 1]
```diff
@@ -32,4 +32,4 @@ def copy_file_to_clipboard(path):
 
         # Set to Clipboard
         clipboard = app.clipboard()
-        clipboard.setMimeData(mime)
+        clipboard.setMimeData(mime)
```
Comment (BigRoy) [inline colorbleed/plugins/global/load/copy_file.py]: Blegh, is this pycharm doing that? :) Shouldn't it actually force a new line being there?

File: colorbleed/__init__.py [hunk 2]
```diff
@@ -24,3 +24,8 @@ def uninstall():
     print("Deregistering global plug-ins..")
     pyblish.deregister_plugin_path(PUBLISH_PATH)
     avalon.deregister_plugin_path(avalon.Loader, LOAD_PATH)
+
+
+def register_loader_plugins():
```
Comment (BigRoy) [inline colorbleed/__init__.py]: Is this needed for this particular PR? Seems like a remainder from something else?

File: colorbleed/plugins/maya/publish/collect_renderlayers.py [hunk 3]
```diff
@@ -154,7 +152,33 @@ def parse_options(self, render_globals):
...
+        # Check for specific pools
+        if "primaryPool" in attributes:
+            pool_a = attributes["primaryPool"]
+            pool_b = attributes["secondaryPool"]
+
+        else:
+            # Backwards compatibility
+            pool_str = attributes.get("pools", None)
+            if pool_str:
+                print("XXX", pool_str.split(";"))
```
Comment (BigRoy) [inline colorbleed/plugins/maya/publish/collect_renderlayers.py]: Debug print statement should be removed?

**Comment by BigRoy** [comment]
Just some minor comments, but functionality seems to be good outside of those. Nice!

File: colorbleed/__init__.py (hunk 2 above)
Comment (aardschok) [reply colorbleed/__init__.py]: This is a remainder of the standalone plugins which we have discussed in PLN-105

**Pull request merged**